Makefile

```diff
@@ -14,21 +14,20 @@
 REPOSITORY := quay.io/prometheus
 NAME       := golang-builder
 VARIANTS   ?= base main
-LATEST     ?= 1.16
-VERSIONS   ?= 1.15 1.16
+VERSION    ?= 1.16
 
 all: build
 
 build:
-	@./build.sh "$(VERSIONS)" "$(VARIANTS)"
+	@./build.sh "$(VERSION)" "$(VARIANTS)"
 
 tag:
-	docker tag "$(REPOSITORY)/$(NAME):$(LATEST)-main" "$(REPOSITORY)/$(NAME):latest"
-	docker tag "$(REPOSITORY)/$(NAME):$(LATEST)-main" "$(REPOSITORY)/$(NAME):main"
-	docker tag "$(REPOSITORY)/$(NAME):$(LATEST)-main" "$(REPOSITORY)/$(NAME):arm"
-	docker tag "$(REPOSITORY)/$(NAME):$(LATEST)-main" "$(REPOSITORY)/$(NAME):powerpc"
-	docker tag "$(REPOSITORY)/$(NAME):$(LATEST)-main" "$(REPOSITORY)/$(NAME):mips"
-	docker tag "$(REPOSITORY)/$(NAME):$(LATEST)-main" "$(REPOSITORY)/$(NAME):s390x"
+	docker tag "$(REPOSITORY)/$(NAME):$(VERSION)-main" "$(REPOSITORY)/$(NAME):latest"
```

Comment:
We can only do this if VERSION is latest

Reply:
VERSION will always be the latest when tag is called. We only call `make tag` from the publish workflow, not the build workflow. `make all` does not call `tag`.